xml/s4s_setup_cluster.xml

```diff
@@ -17,6 +17,12 @@
   wizard once and can then re-apply the saved configuration without
   user interaction.
  </para>
+ <para>
+  Administrators can now use the &hana;-SR Wizard to configure the &hana;
+  cluster on Azure and only have to define a configuration inside the
```

Comment:
@sknorr @tomschr  Maybe here is needed to clarify a little bit. We have 2 new features, one is the capability of running the module unattended. It applies to any usage, like the most common that is the on-premisses deployments.

The second feature is the Cluster Configuration on Azure, that was implemented and the YaST module identifies automatically when  running on Azure, and configures an extra resource needed on Pacemaker, that is a Netcat service to provide healthprobe for the Azure Load Balancer.